### ISSUE TITLE
Clarification/Cleanup around Licensing on the Release Packages

### ISSUE THREAD

**Issue by dcrdev** [MediaBrowser/Emby#3225]
As per Luke's comments:

> There is no violation because we have the ability to have a re-licensed version, and you can assume any build scripts are based off such a version. What we are looking at is how we can make you all happy by opening some of these things up, while still being able to sustain our development. There have been good ideas mentioned from cayars, dcrdev and others. Those efforts will require development effort and take time to separate various components, it is not something we can simply flip a switch on and release. Thanks.

In  https://github.com/MediaBrowser/Emby/issues/3075

It would be helpful to the end user if the GPL metadata was removed from the rpm and deb packages. Along with clarification that releases are not covered under the GPL on this page https://emby.media/terms.html this would help users in understanding their rights and prevent any situations where they might misuse these releases.

To help people understand the terms in which they are using Emby under - it would also help if there were a specific licensing document attributed to these releases explaining those terms.

**Comment by dcrdev** [comment]
Also in a separate but related question - what are the terms under which the binary blobs within this repo are licensed?

Specifically how do they address situations where people are repackaging Emby for other platforms for example distros you don't support, other containerisation platforms, embedded packages like snap and flatpak for instance?  Right now it's not clear whether people have the right to do this and whether or not that is dependent on them independently stripping binary blobs out?

**Comment by kaithar** [comment]
If people have decided to stop complaining based on flawed interpretations of the licencing, I can give you a pretty straight forward explanation of the correct solution to this kind of problem:

1.  Separate the blobs from the GPL source.  Mostly just because it's easier to deal with things that way.
2.  Either get the code to compile without the blobs or, alternatively, provide code for dummy (read: limited or no functionality) blobs that can be compiled and used in place of the closed source ones.
3.  Apply a clear set of licences to both the blobs and the packages.
4.  List the functionality provided by the blobs, usually in the form of what features are lost without them, and why that the source for them can't be provided.
5.  Attempt to reduce the amount of code in the blobs, to reduce the amount of stuff people can whine about not seeing.

I rather dislike the term "open core", but it might be appropriate here.  Normally it evokes an image involving more "essential" features being restricted than the Premium tier has, but it's a very well established business model.

Code doesn't magically write itself, someone somewhere has to pay for it with time or money and the profit margin of the people writing the code determines how much time they devote to a project.  It's cynical but true, even code written for free in someone's spare time is costing that person some of their finite supply of time to create.  Any time an open source project is asked for something the request is essentially reducible to "Project, please give me some money in exchange for these words."

**Comment by dcrdev** [comment]
> Apply a clear set of licences to both the blobs and the packages.

This is more or less why I opened this issue - just in the short term, update the documentation in the repos and on the website to reflect the licensing situation i.e. part A is GPL, part B is proprietry - you can redistribute is based on condition A or B etc etc...

**Comment by LukePulverenti** [comment]
Yes we'll do this, and we're committed to getting the binaries removed. It will just take a little time. Thanks.

**Comment by dcrdev** [comment]
Any updates on either of these things?

**Comment by LukePulverenti** [comment]
We are still working on removing the blobs from the source. It just takes a bit of time to modularize the components so that we can do this. The last release took a step forward and took care of one of them, and we will continue to chip away with each new release. Thanks.
